Earthfile

```diff
@@ -15,9 +15,13 @@ test:
     # Run unit tests
     RUN mix test
 
+   IF echo $ELIXIR | version ">=1.12" 
```

Comment:
Thanks! I can also pass an env var called $RUN_INSTALLER_TESTS right?

Reply:
yep! The IFs should work like bash ifs